app/User.php

```diff
@@ -230,4 +231,57 @@ public function managedCouples()
     {
         return $this->hasMany(Couple::class, 'manager_id');
     }
+
+    public function getAgeAttribute()
```

Comment:
Method name inconsistency? Any specific reason to use `get` and `Attribute`, but not in others?

Reply:
that is Laravel Model Accessor please see here https://laravel.com/docs/5.8/eloquent-mutators